pages/api/subscribe.ts

```diff
@@ -0,0 +1,127 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import md5 from "md5";
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any
```

Comment:
I really didn't feel typing the whole return request haha. I'll cleanup that in the next PR.